client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx

```diff
@@ -47,7 +50,6 @@ import { CheckIcon } from './check-icon';
 import { getRefundPolicies, getRefundWindows, RefundPolicy } from './refund-policies';
 import type { ResponseCart, ResponseCartProduct } from '@automattic/shopping-cart';
 import type { TranslateResult } from 'i18n-calypso';
-
```

Comment:
Since I mentioned it for that one spacing removal and you fixed it, I figured I'd mark the other places this space was removed.